docusign/1.0/modules/ROOT/pages/docusign-connector-examples.adoc

```diff
@@ -0,0 +1,216 @@
+= DocuSign 1.0 Examples - Mule 4
+
+This example shows how to create a DocuSign envelope. The example contains two flows:
+
+. The first flow provides a template for a user interface.
+. The second flow creates the actual DocuSign envelope.
+
+== Prerequisites
+
+You must have:
+
+* Anypoint Studio
+* Mule runtime engine (Mule) 4.2.1 or later
+
+== Create the First Flow
+
+The first flow provides a template for the user interface. To change the template used in the flow,
+select the *Parse Template* component and go to the *Location* field to update the location
+with the path of the wanted template.
```

Comment:
For some reason "desired" sounds better than "wanted" here. I can't explain why.